internal/testutil/pkgdata.go

```diff
@@ -274,3 +274,35 @@ func MustMakeDeb(entries []TarEntry) []byte {
 	}
 	return data
 }
+
+func REG(mode int64, path, content string) TarEntry {
+	return TarEntry{
+		Header: tar.Header{
+			Typeflag: tar.TypeReg,
+			Name:     path,
+			Mode:     mode,
+		},
+		Content: []byte(content),
+	}
+}
+
+func DIR(mode int64, path string) TarEntry {
+	return TarEntry{
+		Header: tar.Header{
+			Typeflag: tar.TypeDir,
+			Name:     path,
+			Mode:     mode,
+		},
+	}
+}
+
+func LNK(mode int64, path, target string) TarEntry {
+	return TarEntry{
+		Header: tar.Header{
+			Typeflag: tar.TypeSymlink,
+			Name:     path,
+			Mode:     mode,
+			Linkname: target,
+		},
+	}
+}
```

Comment:
Although it is obvious what the shorthands mean when you look at the function code, I think it would be nice to have some comments about what the functions are supposed to do. And what the shorthand stands for.

Reply:
Added function comments.